docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+Machine ID can join with a token or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tctl bots add robot --roles=access
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time. The default for tokens is only 30 minutes.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+  
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token. Note,
+    # the "bot-" prefix is required. The name of the bot you will create is "robot".
+    bot_name: bot-robot
+  
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=access
+  ```
+  </TabItem>
+</Tabs>
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary. The `tbot start` command will
+start running Machine ID in a loop, writing renewable certificates to
+`/var/lib/teleport` and the short-lived certificates your application will use
+to `/opt/machine-id`.
+
+In a production environment you will want to run Machine ID in the background
+using a service manager like systemd. However, in this guide you will run it in
+the foreground to better understand how it works.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tbot start \
+     --data-dir=/var/lib/teleport \
+     --destination-dir=/opt/machine-id \
+     --token=00000000000000000000000000000000 \
```

Comment:
I found the Token UX to be confusing. This token isn't a normal token from `tctl tokens add` but instead a token from `tctl bots add`.  This is more confusing as the result of `tctl tokens ls` calls `Name` a Token, and it doesn't show type for IAM based joining. 

```
Token                            Type             Labels Expiry Time (UTC)
-------------------------------- ---------------- ------ ----------------------------------------------
machine_id                       Bot                     01 Jan 00 00:00 UTC (2562047h47m16.854775807s)
GsctZmPxhr4F9NV2iaZ              Bot                     01 Jan 00 00:00 UTC (2562047h47m16.854775807s)
```

Reply:
We should probably clarify this more, but they are both Bot tokens by design. They're all the same on the backend (give or take some parameters), but `tctl bots add` creates a random token for you if you don't provide one

Reply:
I know this sounds crazy, but I couldn't get tokens generated by `tctl tokens add` to work, only tokens created by `tctl bots add` were able to connect.

Reply:
It doesn't sound crazy at all, `tctl tokens add` creates a different type of token and we expect `Bot` here. At the moment the only ways to create those tokens are via `tctl bots add ...` (for standard token joining) or `tctl create -f ...` on a YAML token with the proper roles set (for IAM joining).

Reply:
@benarent I think you have another ticket for this, so I'll resolve this one.